tools/validators/instance_validator/generate_universe.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import os
+
+def build_universe():
+    """Generates ontology universe.
+
+    Returns:
+        Generated universe object.
+    """
+    # add universe building packages to path
+    sys.path.append(os.path.abspath(os.path.join('..', 'ontology_validator'))) 
+    # add ontology files to path
+    sys.path.append(os.path.abspath(os.path.join('../../../', 'ontology')))
```

Comment:
what would happens if the ontology path was changed. can you add a check here to exist with an error message?

Reply:
Done